projects/Mallard/src/screens/onboarding-screen.tsx

```diff
@@ -45,7 +46,7 @@ const OnboardingConsentScreen = ({
                 }}
                 onContinue={() => {
                     onOpenGdprConsent()
-                    setSetting('hasOnboarded', true)
+                    setHasOnboarded(client, true)
```

Comment:
Is there any way we can have a `useSetHasOnboarded` (or a similar abstraction) that internally uses the client hook and injects this for us?

Reply:
It wouldn't bring much benefit afaik. because you'd still have 2 lines of code, the `useSetHasOnboarded` (instead of `useApolloClient()`) and then the `setHasOnboarded`.

On the other hand, we could make the Apollo instance a global variable. Since we only have 1, I think that'd be reasonable.

Reply:
You would also only have one dependency being imported at the top of the file, instead of two. It's less about the LoC and more about having to combine two dependencies everywhere, where one would suffice. I'm not fixed on it but it seems like a bit less mental overhead in order to use it as a dev.